src/frameworks.sources

```diff
@@ -2051,6 +2051,7 @@ IOS_FRAMEWORKS =         \
 	CoreSpotlight \
 	CoreTelephony \
 	CoreText \
+	DataDetection \
```

Comment:
You need to add the framework to `MACCATALYST_FRAMEWORKS` too if you want these bindings in Mac Catalyst (like your availability attributes seem to indicate).